### PR TITLE
Bugs/693 clean up login

### LINKED ISSUE #693: "Have an invite code" is confusing when you only need to enter an email
**To Reproduce**

1. Go to appointment.day
2. Click on Sign up for the beta
3. See text

**Expected behavior**
The heading talks about exciting reasons why people should really enter their email and continue and what they are about to experience.

**Actual behavior**
It says "Have an invite code? Enter it here", which is confusing if you can only enter an email

**Screenshots**
image

### PR DESCRIPTION
**Pull request by MelissaAutumn**
Fixes #693 and #699 

I've also adjusted the copy on the waiting list. Needs a string update!

### Screenshots
image
image

### CODE REVIEW

File: frontend/src/components/GenericModal.vue [hunk 1]
```diff
@@ -35,6 +41,9 @@ onUnmounted(() => {
 <template>
   <div class="new-design overlay" role="dialog" tabindex="-1" aria-labelledby="title" aria-modal="true">
     <div class="modal">
+        <div class="modal-close" v-if="closable" @click="emits('close')" role="button" aria-labelledby="modal-close-button">
+          <img id="modal-close-button" src="@/assets/svg/icons/close.svg" :alt="t('label.close')" :title="t('label.close')"/>
+        </div>
```
Comment (MelissaAutumn) [inline frontend/src/components/GenericModal.vue:46]: I guess I could make this a button as well 🤔

**Comment by MelissaAutumn** [comment]
Note: This is pending another text adjustment!

File: frontend/src/components/GenericModal.vue (hunk 1 above)
Comment (devmount) [reply frontend/src/components/GenericModal.vue:46]: I agree. Maybe we can use our LinkButton.vue? Or create another component for icon buttons.

**Review by devmount — APPROVED** [review]
Yes, looking good. Thank you!

File: frontend/src/components/GenericModal.vue (hunk 1 above)
Comment (MelissaAutumn) [reply frontend/src/components/GenericModal.vue:46]: Oh right I forgot about that already 😅

**Comment by MelissaAutumn** [comment]
More l10n updates 😄 @devmount

**Review by devmount — APPROVED** [review]
Yes, updated.

**Pull request merged**